source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -431,7 +431,10 @@ and `what's the point of periodic monitoring?`_)
 Client construction
 '''''''''''''''''''
 
-The client's constructor MUST NOT do any I/O.
+Except for `initial DNS seed list discovery
```

Comment:
I think "seedlist" can be one word, since it's the title of the spec, but up to you.

Reply:
Looking at this again, the phrase "seed list" occurs a number of times in the sdam spec, but "seedlist" does not occur at all.